addons/addon-base-raas/packages/base-raas-services/lib/data-egress/data-egress-service.js

```diff
@@ -83,9 +83,13 @@ class DataEgressService extends Service {
     } catch (error) {
       throw this.boom.notFound(`Error in fetch egress store info: ${JSON.stringify(error)}`, true);
     }
+
+    if (egressStoreScanResult.length === 0) {
+      return null;
+    }
```

Comment:
Do we need a unit test here?

Reply:
tests added in commit: 2a80047064bb961f9e8b9fa14643cb65033161f7